### PR TITLE
fixes slapfights

### PR DESCRIPTION
**Pull request by MMMiracles**
the fact this wasn't already a thing is a travesty

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Do this for organic limbs too

**Comment by MMMiracles** [comment]
i did, unless theres another set of limbs

File: code/modules/surgery/bodyparts/bodyparts.dm [hunk 1]
```diff
@@ -299,6 +300,7 @@
 	desc = "Over 87% of humans are right handed. That figure is much lower \
 		among humans missing their right arm."
 	icon_state = "r_arm"
+	attack_verb = list("slapped", "punched")
```
Comment (KazeEspada) [inline code/modules/surgery/bodyparts/bodyparts.dm]: No fisted?

Comment (MMMiracles) [inline code/modules/surgery/bodyparts/bodyparts.dm]: erp is banned

Comment (Bawhoppen) [inline code/modules/surgery/bodyparts/bodyparts.dm]: seperation is absolute though

Comment (Cheridan) [inline code/modules/surgery/bodyparts/bodyparts.dm]: its codebanned too

**Comment by MMMiracles** [comment]
PRs so good they get in early

**Pull request merged**